internal/components/trigger/http.go

```diff
@@ -79,7 +79,8 @@ func (h *httpAction) Do() chan error {
 			case <-t.C:
 				err := h.execute()
 
-				// `h.times == h.executedCount` makes sure to only send the first error
+				// `send == false && h.times == h.executedCount` makes sure to only send firstly the error and
```

Comment:
What is the `send firstly the error`? Is this?

```suggestion
				// `send == false && h.times == h.executedCount` makes sure to only send the error firstly and
```